### PR TITLE
CLDR-11982 lv collation

### PR DESCRIPTION
**Pull request by Mazurs**
Use the collation rules described in the LVS 24:1993 standard.

CLDR-11982

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by srl295 — COMMENTED** [review]
structure lgtm

**Comment by srl295** [comment]
@Mazurs thanks for creating this (and signing CLA!) — i'll kick off the autotests, meanwhile @markusicu can review…

**Comment by srl295** [comment]
@Mazurs looks like it passes tests.  give us some time to take a look, but this seems like it should be on it way to merging.

File: common/collation/lv.xml [hunk 1]
```diff
@@ -13,15 +13,19 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<collations  >
 			<collation type="standard" references="Dzintra Kalniņa: Latvian-English Dictionary ISBN 9984-700-52-6">
```
Comment (markusicu) [inline common/collation/lv.xml:14]:
```suggestion
			<collation type="standard" references="LVS 24:1993, LATVIAN LANGUAGE SUPPORT FOR COMPUTERS">
```

**Review by markusicu — CHANGES_REQUESTED** [review]
the order looks like the provided translated reference

please change the reference in the data file

**Review by markusicu — COMMENTED** [review]
The changes all look good now, but the second commit is missing the `CLDR-11982 ` prefix, and that blocks merging the PR.

Please
- either squash the commits and use just the first commit message
- or amend the second commit, adding the prefix to the commit message (and I can then squash-and-merge)

**Comment by Mazurs** [comment]
Made another pull request with merged commits: #3469

**Comment by markusicu** [comment]
> Made another pull request with merged commits: #3469

Ok... you could have just amended or squashed this one... doesn't really matter, I approved the new one and will merge if the tests are happy.

**Pull request closed without merging**